app/views/api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,9 @@
+### 16th June 2020
+
+Sandbox changes:
+
+- Sandbox no longer sends emails to providers about application state changes. The email history of each application is available under an "Emails" tab on the web interface.
```

Comment:
Perhaps leave this out, if this tab isn't going to be permanent.

```suggestion
- Sandbox no longer sends emails to providers about application state changes.
```

Reply:
Sure